### PR TITLE
docs: add more info in contributing

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: docs/zh/developer/contributing.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
-# Contributing
+# 代码贡献
 Please refer to [Contribution Guideline](https://github.com/4paradigm/OpenMLDB/blob/main/CONTRIBUTING.md)
 
+## Pull Request须知
```
Comment (lumianph) [inline docs/zh/developer/contributing.md]: => Pull Request (PR)
then change all "pr" in this doc to "PR"; 
or you may just use the full name pull request, which is better

Comment (lumianph) [reply docs/zh/developer/contributing.md]: later we may add a section to our CONTRIBUTING.md, then also mention this guideline in the PR template

Comment (vagetablechicken) [reply docs/zh/developer/contributing.md]: fix pr typo done

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**